<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="description" content="This package contains Table S2 from Magini et.al. (2019). This supplimentary data describes the clinical phenotype of patients included in the study. The data has been transformed into an analysis ready dataset in tidy format. ">
<title>Summary of SMPD4-Related Clinical Phenotype • SMPD4</title>
<script src="deps/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<link href="deps/bootstrap-5.1.3/bootstrap.min.css" rel="stylesheet">
<script src="deps/bootstrap-5.1.3/bootstrap.bundle.min.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- bootstrap-toc --><script src="https://cdn.rawgit.com/afeld/bootstrap-toc/v1.0.1/dist/bootstrap-toc.min.js"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- search --><script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/6.4.6/fuse.js" integrity="sha512-zv6Ywkjyktsohkbp9bb45V6tEMoWhzFzXis+LrMehmJZZSys19Yxf1dopHx7WzIKxr5tK2dVcYmaCk2uqdjF4A==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/autocomplete.js/0.38.0/autocomplete.jquery.min.js" integrity="sha512-GU9ayf+66Xx2TmpxqJpliWbT5PiGYxpaG8rfnBEk1LL8l1KGkRShhngwdXK1UgqhAzWpZHSiYPc09/NwDQIGyg==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js" integrity="sha512-5CYOlHXGh6QpOFA/TeTylKLWfB3ftPsde7AnmhuitiTX4K5SqCLBeKro6sPS8ilsz1Q4NRx3v8Ko2IBiszzdww==" crossorigin="anonymous"></script><!-- pkgdown --><script src="pkgdown.js"></script><meta property="og:title" content="Summary of SMPD4-Related Clinical Phenotype">
<meta property="og:description" content="This package contains Table S2 from Magini et.al. (2019). This supplimentary data describes the clinical phenotype of patients included in the study. The data has been transformed into an analysis ready dataset in tidy format. ">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body>
    <a href="#main" class="visually-hidden-focusable">Skip to contents</a>
    

    <nav class="navbar fixed-top navbar-light navbar-expand-lg bg-light"><div class="container">
    
    <a class="navbar-brand me-2" href="index.html">SMPD4</a>

    <small class="nav-text text-muted me-auto" data-bs-toggle="tooltip" data-bs-placement="bottom" title="">0.0.0.9000</small>

    
    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbar" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
      <span class="navbar-toggler-icon"></span>
    </button>

    <div id="navbar" class="collapse navbar-collapse ms-3">
      <ul class="navbar-nav me-auto">
<li class="nav-item">
  <a class="nav-link" href="reference/index.html">Reference</a>
</li>
      </ul>
<form class="form-inline my-2 my-lg-0" role="search">
        <input type="search" class="form-control me-sm-2" aria-label="Toggle navigation" name="search-input" data-search-index="search.json" id="search-input" placeholder="Search for" autocomplete="off">
</form>

      <ul class="navbar-nav"></ul>
</div>

    
  </div>
</nav><div class="container template-home">
<div class="row">
  <main id="main" class="col-md-9"><div class="section level1">
<div class="page-header"><h1 id="smpd4">SMPD4<a class="anchor" aria-label="anchor" href="#smpd4"></a>
</h1></div>
<!-- badges: start -->
<p>The goal of SMPD4 is to provide analysis ready data sets on SMPD4 data.</p>
<p>Sphingomyelin Phosphodiesterase 4 (SMPD4) is a type of sphinogmyelinase protein encoded by the SMPD4 gene. Patients with a bi-allelic loss of function variants in this gene are associated with a condition known as neurodevelopmental disorder with microcephaly, arthrogryposis, and structural brain anomalies (NEDMABA; <a href="https://omim.org/entry/618622" class="external-link">618622</a>).</p>
<p>Few cases exist in the scientific literature with <a href="https://doi.org/10.1016/j.ajhg.2019.08.006" class="external-link">Magini et al. (2019)</a> reporting 23 members of 12 unrelated families. The genotype-phenotype relationship of NEDMABA is heterogeneous and aside from case-reports, little data is available for statistical analysis. Magini et al. (2019) offer a supplementary table with summarised clinical phenotype data of the included subjects. Unfortunately these data are non standardised, heterogeneous and largely textual based. This project aims to standardise and process this data into tidy format <a href="https://doi.org/10.1016/j.ajhg.2019.08.006" class="external-link">(Wickham (2014))</a> so it is analysis-ready and extensible for other researchers to better understand the relationship between clinical factors discovered in the literature.</p>
<div class="section level2">
<h2 id="installation">Installation<a class="anchor" aria-label="anchor" href="#installation"></a>
</h2>
<p>You can install the development version of SMPD4 from <a href="https://github.com/" class="external-link">GitHub</a> with:</p>
<div class="sourceCode" id="cb1"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co"># install.packages("devtools")</span></span>
<span><span class="fu">devtools</span><span class="fu">::</span><span class="fu">install_github</span><span class="op">(</span><span class="st">"deanmarchiori/SMPD4"</span><span class="op">)</span></span></code></pre></div>
</div>
<div class="section level2">
<h2 id="example">Example<a class="anchor" aria-label="anchor" href="#example"></a>
</h2>
<div class="sourceCode" id="cb2"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://deanmarchiori.github.io/SMPD4">SMPD4</a></span><span class="op">)</span></span>
<span></span>
<span><span class="fu"><a href="https://rdrr.io/r/utils/data.html" class="external-link">data</a></span><span class="op">(</span><span class="st">"smpd4_phenotype"</span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/utils/head.html" class="external-link">head</a></span><span class="op">(</span><span class="va">smpd4_phenotype</span><span class="op">[</span>,<span class="fl">1</span><span class="op">:</span><span class="fl">20</span><span class="op">]</span><span class="op">)</span></span>
<span><span class="co">#&gt;    id family individual deceased survival_time variant_type locus_1 locus_2</span></span>
<span><span class="co">#&gt; 1 1_1      1          1     TRUE            30   homozygote    1320      NA</span></span>
<span><span class="co">#&gt; 2 1_2      1          2     TRUE            60   homozygote    1320      NA</span></span>
<span><span class="co">#&gt; 3 1_3      1          3     TRUE            30   homozygote    1320      NA</span></span>
<span><span class="co">#&gt; 4 1_4      1          4     TRUE          1095   homozygote    1320      NA</span></span>
<span><span class="co">#&gt; 5 2_1      2          1     TRUE          2190   homozygote    1483      NA</span></span>
<span><span class="co">#&gt; 6 2_2      2          2    FALSE          1825   homozygote    1483      NA</span></span>
<span><span class="co">#&gt;   gender ethnicity consanguineous termination birth_gestation</span></span>
<span><span class="co">#&gt; 1 female   Turkish           TRUE       FALSE              39</span></span>
<span><span class="co">#&gt; 2   male   Turkish           TRUE       FALSE              37</span></span>
<span><span class="co">#&gt; 3   male   Turkish           TRUE       FALSE              36</span></span>
<span><span class="co">#&gt; 4 female   Turkish           TRUE       FALSE              38</span></span>
<span><span class="co">#&gt; 5 female   Morocco           TRUE       FALSE              37</span></span>
<span><span class="co">#&gt; 6   male   Morocco           TRUE       FALSE              37</span></span>
<span><span class="co">#&gt;   route_vaginal_vs_c_sect birth_weight birth_ofc birth_length age_at_demise</span></span>
<span><span class="co">#&gt; 1                     SVD         2175      30.5           NA            30</span></span>
<span><span class="co">#&gt; 2                      CS         1860      31.0           NA            60</span></span>
<span><span class="co">#&gt; 3                      CS         1955      30.0           NA             2</span></span>
<span><span class="co">#&gt; 4                      CS         2780      30.5           41          1095</span></span>
<span><span class="co">#&gt; 5                      CS         2045      29.0           39          2190</span></span>
<span><span class="co">#&gt; 6                      CS         2300        NA           NA            NA</span></span>
<span><span class="co">#&gt;   age_at_last_follow_up microcephaly_ind</span></span>
<span><span class="co">#&gt; 1                    30                1</span></span>
<span><span class="co">#&gt; 2                    60                1</span></span>
<span><span class="co">#&gt; 3                    30                1</span></span>
<span><span class="co">#&gt; 4                  1095                1</span></span>
<span><span class="co">#&gt; 5                    NA                1</span></span>
<span><span class="co">#&gt; 6                  1825                1</span></span></code></pre></div>
</div>
<div class="section level2">
<h2 id="references">References<a class="anchor" aria-label="anchor" href="#references"></a>
</h2>
<p>Magini, P., Smits, D. J., Vandervore, L., Schot, R., Columbaro, M., Kasteleijn, E., … &amp; Mancini, G. M. (2019). Loss of SMPD4 causes a developmental disorder characterized by microcephaly and congenital arthrogryposis. The American Journal of Human Genetics, 105(4), 689-705. <a href="https://doi.org/10.1016/j.ajhg.2019.08.006" class="external-link uri">https://doi.org/10.1016/j.ajhg.2019.08.006</a></p>
<p>Wickham, H. . (2014). Tidy Data. Journal of Statistical Software, 59(10), 1–23. <a href="https://doi.org/10.18637/jss.v059.i10" class="external-link uri">https://doi.org/10.18637/jss.v059.i10</a></p>
</div>
</div>
<div class="section level1">
<h1 id="data-use-and-licencing">Data use and licencing<a class="anchor" aria-label="anchor" href="#data-use-and-licencing"></a>
</h1>
<p>These data are Copyright 2019 American Society of Human Genetics.</p>
<p>Data is used under Elsevier user license <a href="https://www.elsevier.com/about/policies/open-access-licenses/elsevier-user-license" class="external-link uri">https://www.elsevier.com/about/policies/open-access-licenses/elsevier-user-license</a></p>
<p>Use permitted for read, download, text and data mining. Used for Non-Commercial purposes.</p>
<p>All outputs of this R package are data mining related for non-commerical use.</p>
<p>DOI: <a href="https://doi.org/10.1016/j.ajhg.2019.08.006" class="external-link uri">https://doi.org/10.1016/j.ajhg.2019.08.006</a></p>
</div>

  </main><aside class="col-md-3"><div class="license">
<h2 data-toc-skip>License</h2>
<ul class="list-unstyled">
<li>file <a href="LICENSE-text.html">LICENSE</a>
</li>
</ul>
</div>


<div class="citation">
<h2 data-toc-skip>Citation</h2>
<ul class="list-unstyled">
<li><a href="authors.html#citation">Citing SMPD4</a></li>
</ul>
</div>

<div class="developers">
<h2 data-toc-skip>Developers</h2>
<ul class="list-unstyled">
<li>Dean Marchiori <br><small class="roles"> Author, maintainer </small>  </li>
<li><a href="authors.html">More about authors...</a></li>
</ul>
</div>

<div class="dev-status">
<h2 data-toc-skip>Dev status</h2>
<ul class="list-unstyled">
<li><a href="https://lifecycle.r-lib.org/articles/stages.html#experimental" class="external-link"><img src="https://img.shields.io/badge/lifecycle-experimental-orange.svg" alt="Lifecycle: experimental"></a></li>
</ul>
</div>

  </aside>
</div>


    <footer><div class="pkgdown-footer-left">
  <p></p>
<p>Developed by Dean Marchiori.</p>
</div>

<div class="pkgdown-footer-right">
  <p></p>
<p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.0.6.</p>
</div>

    </footer>
</div>

  

  

  </body>
</html>
